go.mod

```diff
@@ -25,12 +26,9 @@ require (
 )
 
 replace (
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
 	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190814194116-a94e914914f4+incompatible
-	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20190813201236-5a5508328169
```

Comment:
We don't need OpenShift's client any more?

Reply:
The IDE removed it on its own, so I guess not? It's compiling alright. Should we check anything else?

`grep -r 'github.com/openshift/client-go' *` returns nothing as well.

Reply:
Removing dependencies is good. :sunglasses: